### PR TITLE
Change "Ansible playbook" to "Ansible Playbook"

### PR DESCRIPTION
**Pull request by apinnick**
Closes https://github.com/theforeman/foreman-documentation/issues/3128

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/con_api-overview.adoc [hunk 1]
```diff
@@ -2,15 +2,15 @@
 = {Project} API overview
 
 You can write custom scripts and external applications that access the {Project} API over HTTP with the Representational State Transfer (REST) API provided by {ProjectServer}.
-Use the REST API to integrate with enterprise IT systems and third-party applications, perform automated maintenance or error checking tasks, and automate repetitive tasks with scripts. 
+Use the REST API to integrate with enterprise IT systems and third-party applications, perform automated maintenance or error checking tasks, and automate repetitive tasks with scripts.
```
Comment (asteflova) [inline guides/common/modules/con_api-overview.adoc:5]: This line doesn't change the capitalization. Did you include this intentionally?

Comment (apinnick) [reply guides/common/modules/con_api-overview.adoc:5]: No, that's just the 'Remove white space" setting in VSCode. Some of my projects require it to be enabled.

**Review by Lennonka — APPROVED** [review]
LGTM. I don't mind about the removed whitespace.
Unless there are further comments, I'll merge it tomorrow.

**Review by maximiliankolb — APPROVED** [review]
diff LGTM; please add it to https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization

**Comment by apinnick** [comment]
> diff LGTM; please add it to https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization

@maximiliankolb  
https://github.com/theforeman/foreman-documentation/pull/3134

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked to "3.11":
   d2340c9574..49f531c922  3.11 -> 3.11

**Pull request merged**